.NET/Microsoft.Recognizers.Text.DateTime/Extractors/CJK/BaseCJKDatePeriodExtractor.cs

```diff
@@ -35,12 +35,19 @@ public List<ExtractResult> Extract(string text)
 
         public List<ExtractResult> Extract(string text, DateObject referenceTime)
         {
+            // Normalize input
+            if (this.config.NormalizeCharMap != null)
```

Comment:
This should be happening before it reaches here. Why is it not? Otherwise it will be called multiple times and also on other sub-types.

Reply:
And does each CJK language needs its own map? Can't one be shared?

Reply:
That is true, here we followed the same approach used in Japanese Date. 
I was thinking we could move it to MergedExtractor, so it would be called only once. 
But is not it that a user can also call each sub-entity separately? In that case the normalization would be missed.

Reply:
For the moment only Japanese needs normalization. For Chinese and Korean the map is null.

Reply:
ah, I found what you mean, the normalization already occurs at the model level. So for the various sub-entities I will only add it to the corresponding TestBase methods.

Reply:
updated